packages/coinstac-ui/app/render/components/consortium/consortium-pipeline.jsx

```diff
@@ -1,53 +1,106 @@
-import React from 'react';
-import { Button, Col, DropdownButton, MenuItem, Row, Well } from 'react-bootstrap';
+import React, { Component } from 'react';
+import { Button, Col, DropdownButton, MenuItem, Nav, NavItem, Row, Well } from 'react-bootstrap';
 import { LinkContainer } from 'react-router-bootstrap';
+import { graphql } from 'react-apollo';
+import ApolloClient from '../../state/apollo-client';
+import PropTypes from 'prop-types';
+import { fetchAllConsortiaFunc, fetchAllPipelinesFunc, saveActivePipelineFunc } from '../../state/graphql/functions';
 
 const styles = {
   activePipelineButton: { margin: '0 5px 0 0' },
   activePipelineParagraph: { borderBottom: '1px solid black' },
   textCenter: { textAlign: 'center' },
 };
 
-const ConsortiumPipeline = () =>
-  (
-    <div>
+class ConsortiumPipeline extends Component{
+  constructor(props) {
+    super(props);
+
+    let ownedPipelines = null;
+    let sharedPipelines = null;
+    let activePipelineRef = {};
+
+    const data = ApolloClient.readQuery({ query: fetchAllPipelinesFunc });
```

Comment:
Perhaps this is why you loaded the fetchAllPipelinesFunc on the list page? The problem with that is that a person could conceivably hit the refresh button on this page and not have the fetchAllPipelinesFunc preloaded anymore to pull from as this is on a completely different route. I'd suggest either creating separate Apollo queries for `ownedPipelines` and `sharedPipelines` or moving the `fetchAllPipelines` call to `consortium-tabs.jsx` since that is the parent of this same route. However, the problem I have with the latter is that I think we'll eventually make all of these tabs their own state, meaning that we'll end up having to refactor it then anyway to load the queries here. Also, maybe it won't finish the query before loading its subcomponents anyway?